### DISCUSSION TITLE
Tauri

### DISCUSSION THREAD

**Discussion by muuvmuuv** [oldj/SwitchHosts#637] [Ideas]
SwitchHosts is nice, thanks for this neat App. Since I am using an M1, I really care a lot more about memory and other stuff. SwitchHosts is using Electron, which already uses a lot of memory and is bloated for the majority of apps. 

I would like to inform you (if you did not already hear about it) that there is a better solution called Tauri. Underneath, it uses Rust as its core and an open-sourced WebView engine. I tried it, and it's mind-blowing fast and takes zero resources. Maybe you can take a look at it and consider it as a replacement for V5.

Let me know what you and others think!

> https://tauri.studio/en/

**Comment by oldj** [comment]
Thanks for the suggestion.

Electron is a bit heavy for SwitchHosts, I have used wxPython, MacGap in previous versions, but they all had some difficult issues to deal with. I had a brief look at the TAURI you mentioned, thanks for the reintroduction, I will look into it again later.
